helm/aladdin-demo/templates/server/deploy.yaml

```diff
@@ -121,21 +125,16 @@ spec:
         volumeMounts:
         - name: workdir
           mountPath: "/work-dir"
-{{ end }} # /nginx.use
-{{ if .Values.redis.create }}
```

Comment:
`redis_check` already has a check for this if case, so just include it .. same for all these 3 other inclusions